services.yaml

```diff
@@ -45,10 +45,44 @@ paths:
     get:
       tags:
         - General
-      description: "List all services"
+      summary: "List all services"
+      description: "List all services accessible by the user"
+      parameters:
+        - in: query
+          name: service_key_name
+          description: Service key name
+          type: string
+          required: false
       responses:
         200:
-          description: TBD
+          description: List of all accessible services
+          schema:
+            properties:
+              services:
+                type: array
+                description: list of services
+                items:
+                  $ref: "#/definitions/ServiceBasic"
+        401:
+          description: Unauthorized
+          $ref: "#/definitions/UnauthorizedError"
+    post:
+      tags:
+        - General
+      summary: Generate a new service key
+      description: "Create a new service"
+      parameters:
+        - in: body
+          key_name: service
```

Comment:
not sure what is `key_name` and `example` in this space. shouldn't this be a JSON?

Reply:
fixed